src/gui/tracks/TrackView.cpp

```diff
@@ -445,4 +459,12 @@ void TrackView::setIndicatorMute(FadeButton* indicator, bool muted)
 }
 
 
+void TrackView::resizeToHeight(int h)
+{
+	setFixedHeight(qMax<int>(h, MINIMAL_TRACK_HEIGHT));
+		m_trackContainerView->realignTracks();
+		m_track->setHeight(height());
```

Comment:
The indentation of the later lines here doesn't match the first.

Reply:
Fixed with commit ffb2a669.